src/progress-bar.tsx

```diff
@@ -29,17 +30,29 @@ type Props = {
     progressPercent: number;
     color?: string;
     children?: void;
+    'aria-label'?: string;
+    'aria-labelledby'?: string;
 };
 
-const ProgressBar: React.FC<Props> = ({progressPercent, color}) => {
+const ProgressBar: React.FC<Props> = ({
+    progressPercent,
+    color,
+    'aria-label': ariaLabel,
+    'aria-labelledby': ariaLabelledBy,
+}) => {
     const classes = useStyles({progressPercent, color});
-
+    const {texts} = useTheme();
+    const defaultLabel = texts.loading;
+    const label = ariaLabelledBy ? undefined : ariaLabel || defaultLabel;
     return (
         <div
             className={classes.barBackground}
+            role="progressbar"
```

Comment:
new version of axe found this issue. We were using `aria-valuenow` without a proper `role`. Also, a label is needed for this component, so I have included `aria-label` and `aria-labelledby` props and default to `text.loading`